File: WebApp/WebApp/Models/Schedule.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class Schedule
    {
        public int Id { get; set; }
        [Required]
        public string Departure { get; set; }
        public int ScheduleTypeId { get; set; }
        public ScheduleType ScheduleType { get; set; }
        public int LineId { get; set; }
        public Line Line { get; set; }
    }
}

File: FrontEndProjectWeb2/src/app/models/Schedule.ts
import { ScheduleType } from './ScheduleType';
import { Line } from './Line';

export class Schedule{
    Id:number;
    Departure:string = "";
    ScheduleTypeId:number;
    ScheduleType:ScheduleType;
    LineId:number;
    Line:Line;
}